common-design/sass/cd-header/_cd-nav.scss

```diff
@@ -33,30 +50,15 @@
   }
 }
 
-.cd-site-header__nav-label--closed {
-  .open & {
-    display: none;
-  }
-}
-
-.cd-site-header__nav-label--open {
-  display: none;
-
-  .open & {
-    display: block;
-    font-size: 20px;
-  }
-}
-
 .cd-nav {
-  background: $cd-mid-bg-color;
-  height: calc(100vh - 35px);
-  max-width: 350px;
-  padding: 40px 0 36px 0;
+  background: $cd-white-bg-color;
+  height: calc(100vh - (#{$cd-global-header-height} + #{$cd-site-header-height} + 3px));
```

Comment:
👍 to documenting [magic numbers](https://css-tricks.com/magic-numbers-in-css/) with Sass vars!